pkg/server/server.go

```diff
@@ -115,6 +115,7 @@ func (s *Server) Serve() error {
 
 			// protected routes if using auth
 			r.HandleFunc("/generate", s.Generate(opts...))
+			r.HandleFunc("/status", s.GetStatus)
```

Comment:
Do we want this to be in the unprotected routes?

Reply:
Yes.